ts/src/pro/ripio.ts

```diff
@@ -29,58 +27,39 @@ export default class ripio extends ripioRest {
         });
     }
 
-    async watchTrades (symbol: string = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+    async watchTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' watchTicker() requires a symbol argument');
+        }
         await this.loadMarkets ();
         const market = this.market (symbol);
-        symbol = market['symbol'];
-        const name = 'trades';
+        const pair = this.parseSymbol (symbol);
+        const name = 'trade@' + pair;
         const messageHash = name + '_' + market['id'].toLowerCase ();
-        const url = this.urls['api']['ws'] + messageHash + '/' + this.options['uuid'];
+        const url = this.urls['api']['ws'];
         const subscription = {
-            'name': name,
+            'topics': [ name ],
+            'method': 'subscribe',
             'symbol': symbol,
             'messageHash': messageHash,
-            'method': this.handleTrade,
+            'methodToCall': this.handleTrade,
         };
-        const trades = await this.watch (url, messageHash, undefined, messageHash, subscription);
+        const trades = await this.watch (url, messageHash, subscription, messageHash, subscription);
         if (this.newUpdates) {
             limit = trades.getLimit (symbol, limit);
         }
         return this.filterBySinceLimit (trades, since, limit, 'timestamp', true);
     }
 
-    handleTrade (client: Client, message, subscription) {
-        //
-        //     {
-        //         messageId: 'CAAQAA==',
-        //         payload: 'eyJjcmVhdGVkX2F0IjogMTYwMTczNjI0NywgImFtb3VudCI6ICIwLjAwMjAwIiwgInByaWNlIjogIjEwNTkzLjk5MDAwMCIsICJzaWRlIjogIkJVWSIsICJwYWlyIjogIkJUQ19VU0RDIiwgInRha2VyX2ZlZSI6ICIwIiwgInRha2VyX3NpZGUiOiAiQlVZIiwgIm1ha2VyX2ZlZSI6ICIwIiwgInRha2VyIjogMjYxODU2NCwgIm1ha2VyIjogMjYxODU1N30=',
-        //         properties: {},
-        //         publishTime: '2020-10-03T14:44:09.881Z'
-        //     }
-        //
-        const payload = this.safeString (message, 'payload');
+    handleTrade (client, message, subscription) {
```

Comment:
add comment of example message. Add typescript type
```suggestion
    handleTrade (client: Client, message, subscription) {
```